static/templates/index.html

```diff
@@ -35,6 +42,9 @@
     <script type="text/javascript" src="{{root}}/skin/index.js?KIWIXCACHEID" defer></script>
   </head>
   <body>
+    <a href="/catalog/v2/entries" id="rssFeedLink">
+      <img src="{{root}}/skin/rss.png?KIWIXCACHEID" class="rssLogo" alt="Kiwix RSS Feed" aria-label="Kiwix RSS Feed" title="Kiwix RSS Feed">
```

Comment:
Label, alt, title should be "Library OPDS Feed"
Remove "rss" and replace by "feed" everywhere